test/xspec.bats

```diff
@@ -190,3 +190,10 @@ teardown() {
     [ "$status" -eq 0 ]
     [[ "${lines[3]}" =~ "Testing with" ]]
 }
+
+@test "executing the Saxon XProc harness generates a report with UTF-8 encoding" {
+	run bash -c "java -Xmx1024m -cp ${XMLCALABASH_CP} com.xmlcalabash.drivers.Main -isource=xspec-72.xspec xspec-home=file:///home/travis/build/xspec/xspec/ -oresult=/home/travis/build/xspec/xspec/test/xspec/xspec-72-report.html /home/travis/build/xspec/xspec/src/harnesses/saxon/saxon-xslt-harness.xproc && grep 'charset=UTF-8' /home/travis/build/xspec/xspec/test/xspec/xspec-72-report.html"
```

Comment:
Agreed in that aspect. But I'm concerned about the preciseness. Suppose someday someone finds this line

```xml
<x:expect label="the encoding is correct"> 
```

and wonders what the correct means actually. After glancing at the bug record, he would possibly augment the label as follows

```xml
<x:expect label="the encoding is correct; html header has charset=UTF-8">
```

and then the test by `grep` will start to always succeed...